Components/Grid/grid.html

```diff
@@ -0,0 +1,158 @@
+                <div id="line">
+                    <hr class="hr-sideBar" />
+                </div>
+
+                <a class="side-bar-items" href="../Avatar/avatar.html">Avatar</a>
+                <a class="side-bar-items " href="../Alert/alert.html">Alert</a>
+                <a class="side-bar-items " href="../Badge/badge.html">Badge</a>
+                <a class="side-bar-items " href="../Button/button.html">Button</a>
+                <a class="side-bar-items" href="../Cards/card.html">Card</a>
+                <a class="side-bar-items" href="../Responsive-Image//image.html">Image</a>
+                <a class="side-bar-items " href="../Input/input.html">Input</a>
+
+                <a class="side-bar-items 
+" href="../List/list.html">List</a>
+                <a class="side-bar-items" href="../Toast/toast.html">Toast</a>
+                <a class="side-bar-items " href="../Rating/rating.html">Rating</a>
+                <a class="side-bar-items active" href="../Grid/grid.html">Grid</a>
+
+            </div>
+
+            <div class="content">
+                <span class="title">Grid</span>
+                <p class="margin-bottom-1 p-content">Use grid when you want sections in view. Check below some examples
+                    of side by side cards.
+
+                </p>
+                <hr class="margin-bottom-1">
+                <!-- grid
+ -->
+
+                <br></br>
+
+                <div class="grid-container">
+
+                    <div class="grid">
+                        <div class="child">1</div>
+                        <div class="child">2</div>
+                        <div class="child">3</div>
+                        <div class="child">4</div>
+                        <div class="child">5</div>
+                        <div class="child">6</div>
+                    </div>
+
+                </div>
+
+
+                <iframe class="iframeHide" src="https://carbon.now.sh/embed/CoPSILqk7DNcqb7YW0by"
+                    style="width: 411px; height: 335px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+
+                <hr class="margin-bottom-1">
+
+                <!-- grid
+ -->
+
+                <br></br>
+
+                <div class="grid-container">
+
+                    <div class="grid-2-items">
+                        <div class="child">1</div>
+                        <div class="child">2</div>
+                        <div class="child">3</div>
+                        <div class="child">4</div>
+                        <div class="child">5</div>
+                        <div class="child">6</div>
+                    </div>
+
+                </div>
+
+
+
+                <iframe class="iframeHide" src="https://carbon.now.sh/embed/NJKkJM9eaeXAenJsN7TI"
+                    style="width: 403px; height: 353px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
suggestion: you can use a class to style iframe to maintain uniformity

Reply:
Thanks for the review [dhimanaishwarya](https://github.com/dhimanaishwarya)